neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -29,9 +29,9 @@ public abstract class Transaction : IEquatable<Transaction>, IInventory
 
         public readonly TransactionType Type;
         public byte Version;
-        public TransactionAttribute[] Attributes;
-        public CoinReference[] Inputs;
-        public TransactionOutput[] Outputs;
+        public TransactionAttribute[] Attributes = new TransactionAttribute[0];
```

Comment:
It's better because it's static

```suggestion
        public TransactionAttribute[] Attributes = Array.Empty<TransactionAttribute>();
```

Reply:
Fixed